app/jobs/reports/identity_reuse_report.rb

```diff
@@ -0,0 +1,51 @@
+require 'identity/hostdata'
+
+module Reports
+  class IdentityReuseReport < BaseReport
+    REPORT_NAME = 'credential-reuse-report'.freeze
+
+    include GoodJob::ActiveJobExtensions::Concurrency
+
+    good_job_control_concurrency_with(
+      total_limit: 1,
+      key: -> { "#{REPORT_NAME}-#{arguments.first}" },
+    )
+
+    def perform(_date)
```

Comment:
writing the report this way means we can't backfill and check historical data or trends.

What if we added `AND created_at < #{date}` to the queries, so that we can estimate past values as well and backfill?

Reply:
You can't have a range for the queries because then it will count newly created ones as not reused.

Reply:
if we do a one-sided comparison (`created_at < date`) then I think we can approximate what the value would have been at that previous point in time, I think it would be valuable to be able to compare farther back than just the dates we have run this report since

Reply:
Ah yes.  A one sided comparison will work.  I'll add that.